doc/config.txt

```diff
@@ -0,0 +1,22 @@
+Synopsis
+
+    cordova-cli config <command> [options]
+    
+
+The config command can be used to set, get, delete, edit, and list the 
+contents of the user files.
```

Comment:
"user files" here was confusing to me - it was unclear to me what that is.

Based on the [cordova-discuss proposal](https://github.com/cordova/cordova-discuss/pull/63/commits/788be4b3a2f360cdc2c5b4ddd46d7547a7f45416), we want to model this command after `npm config`, and its synopsis goes like this:

>        npm-config - Manage the npm configuration files

`git config` serves a similar purpose, I think, and its synopsis is:

>        git-config - Get and set repository or global options

One suggestion: "set, get, delete, edit, and list global cordova options" ?